python/lsst/ts/IntegrationTests/mt_state_transition_configs.py

```diff
@@ -0,0 +1,140 @@
+# This file is part of ts_IntegrationTests.
```

Comment:
I'd call this maintel_state.. for clarity.

Reply:
Fixed